app/scripts/context/config-context.tsx

```diff
@@ -0,0 +1,27 @@
+import React, { createContext } from 'react';
+
+interface VedauiConfig {
+  mapboxToken: string;
+  envApiStacEndpoint: string;
+  envApiRasterEndpoint: string;
```

Comment:
⛏️ : the endpoints are prefixed with `env` - maybe `envMapboxToken` to keep the standard 🤷🏼‍♀️